### ISSUE TITLE
Draw Borders On Top: Medium After Load (1.9.0.0.10517)

### ISSUE THREAD

**Issue by Cernelius** [triplea-game/triplea#3525]
When you load a map with a zoom set lower than 100 (you need to have loaded a game of that map previously and set the zoom lower than 100), the borders are drawn medium.

I advise the borders always being drawn "low", regardless of the zoom setting (the borders should always stay "low", unless differently set by the user).

To test this matter it is convenient using a map having far from black relief above the borders; for example, "House of Habsburg".

**Comment by Cernelius** [comment]
Test you can do with "House of Habsburg" or "World At War":

- Load the game at zoom 100% (you should have this zoom if you never opened any games of this map before).
- Verify "Draw Borders On Top" being "Low".
- Zoom the map at 50%.
- Verify "Draw Borders On Top" being "Low".
- Close the game.
- Load the game (now at zoom 50%).
- Verify "Draw Borders On Top" being (now) "Medium".

I assume this behaviour is not intended (anymore), if it is I advise not to be, and should be corrected (not setting "Draw Borders On Top" to "Medium" when loading a game of a map zoomed at less than 100%, or ever).

**Comment by Cernelius** [comment]
This issue is outdated.

This behaviour was present after the zoom changes, that I believe were reverted. It should not be present anymore, since that revert, and I tested it that it doesn't.

That ugly thing of putting the borders on top when going under 100% is still there, and it is degrading for maps not having black borders everywhere (like House of Habsburg, for all borders, or World At War, for the sea zones to sea zones borders only), and I may open a "fix request" about that, but mostly wondering if @RoiEXLab is getting back to the zoom that was reverted (as his improved zoom didn't need this behaviour anyways).
